contracts/BancorEtherToken.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.10;
```

Comment:
Any reason why you changed all file names from lowercase + underscore to camel case? Is there a best practice in solidity for that?